### PR TITLE
Add struct field description

### PR DESCRIPTION
**Pull request by glebtv**
**Describe the PR**
Support parsing struct field descriptions from godoc comments

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -440,12 +440,19 @@ func (parser *Parser) parseStruct(pkgName string, field *ast.Field) (properties
 	if structField.name == "" {
 		return
 	}
+	var desc string
+	if field.Doc != nil {
+		desc = field.Doc.Text()
```
Comment (easonlin404) [inline parser.go]: Can you add tests for it?

**Comment by glebtv** [comment]
Added test.

**Comment by easonlin404** [comment]
@pei0804 any thoughts on this?

**Comment by pei0804** [comment]
Oh sorry.
I forgot.

**Comment by pei0804** [comment]
LGTM

**Comment by pei0804** [comment]
I seemed to be good, so I merged, if did you have a problem?
@easonlin404

**Comment by easonlin404** [comment]
@glebtv @pei0804  update doc or add example would be good otherwise, LGTM. Thanks for your contributions.

**Comment by pei0804** [comment]
The last newline if it is to give a problem. But, I think it is not a big problem.

**Pull request merged**